PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PayPalCreditButton.kt

```diff
@@ -118,7 +118,7 @@ enum class PayPalCreditButtonColor(
                 BLACK.value -> BLACK
                 GOLD.value -> GOLD
                 WHITE.value -> WHITE
-                else -> throw createFormattedIllegalArgumentException("PaymentButtonSize", values().size)
+                else -> throw createFormattedIllegalArgumentException("PayPalCreditButtonColor", values().size)
```

Comment:
This appears to be a typo. Cleaning up in this PR even though its technically unrelated.